gradle-plugin/src/main/java/org/openapitools/openapistylevalidator/gradle/OpenAPIStyleValidatorTask.java

```diff
@@ -0,0 +1,186 @@
+import org.gradle.api.GradleException;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+import org.openapitools.empoa.swagger.core.internal.SwAdapter;
+import org.openapitools.openapistylevalidator.OpenApiSpecStyleValidator;
+import org.openapitools.openapistylevalidator.ValidatorParameters;
+import org.openapitools.openapistylevalidator.ValidatorParameters.NamingConvention;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+
+import java.util.List;
+
+public class OpenAPIStyleValidatorTask extends DefaultTask {
+
+    public static final String INPUT_FILE = "inputFile";
+
+    public static final String VALIDATE_INFO_LICENSE = "validateInfoLicense";
+    public static final String VALIDATE_INFO_DESCRIPTION = "validateInfoDescription";
+    public static final String VALIDATE_INFO_CONTACT = "validateInfoContact";
+
+    public static final String VALIDATE_OPERATION_OPERATION_ID = "validateOperationOperationId";
+    public static final String VALIDATE_OPERATION_DESCRIPTION = "validateOperationDescription";
+    public static final String VALIDATE_OPERATION_TAG = "validateOperationTag";
+    public static final String VALIDATE_OPERATION_SUMMARY = "validateOperationSummary";
+
+    public static final String VALIDATE_MODEL_PROPERTIES_EXAMPLE = "validateModelPropertiesExample";
+    public static final String VALIDATE_MODEL_NO_LOCAL_DEF = "validateModelNoLocalDef";
+
+    public static final String VALIDATE_NAMING = "validateNaming";
+    public static final String IGNORE_HEADER_X_NAMING = "ignoreHeaderXNaming";
+    public static final String PATH_NAMING_CONVENTION = "pathNamingConvention";
+    public static final String PARAMETER_NAMING_CONVENTION = "parameterNamingConvention";
+    public static final String PROPERTY_NAMING_CONVENTION = "propertyNamingConvention";
+
+    private String inputFile;
+
+    private boolean validateInfoLicense = true;
+    private boolean validateInfoDescription = true;
+    private boolean validateInfoContact = true;
+
+    private boolean validateOperationOperationId = true;
+    private boolean validateOperationDescription = true;
+    private boolean validateOperationTag = true;
+    private boolean validateOperationSummary = true;
+
+    private boolean validateModelPropertiesExample = true;
+    private boolean validateModelNoLocalDef = true;
+
+    private boolean validateNaming = true;
+    private boolean ignoreHeaderXNaming = true;
+    private NamingConvention pathNamingConvention = NamingConvention.HyphenCase;
+    private NamingConvention parameterNamingConvention = NamingConvention.CamelCase;
+    private NamingConvention propertyNamingConvention = NamingConvention.CamelCase;
+    
+    public OpenAPIStyleValidatorTask() {
+        this.setGroup("Verification");
+        this.setDescription("Validate that OpenAPI files against style rules");
+    }
+
+    @TaskAction
+    public void execute() {
+        if(inputFile == null) {
```

Comment:
I would say that it should be `if (` instead of `if(` because it is the standard I'm use to. If you do a reformat of the file in your IDE, what is the outcome?